src/certs.rs

```diff
@@ -164,14 +165,14 @@ pub fn write_cert(
     Ok(())
 }
 
-#[derive(StructOpt)]
+#[derive(Parser)]
 pub struct CertificatesArgs {
-    #[structopt(help = "Specify encoding to use for certificates. [PEM | DER]")]
+    /// Specify encoding to use for certificates.
+    #[arg(value_name = "encoding", required = true)]
     pub encoding: CertFormat,
 
-    #[structopt(
-        help = "Directory to store certificates in. Required if requesting an extended-report."
-    )]
+    /// Directory to store certificates in. Required if requesting an extended-report.
+    #[arg(value_name = "certs-dir", required = true)]
     pub certs_dir: PathBuf,
 }
```

Comment:
```suggestion
    /// Directory to store certificates in. Required if requesting an extended-report.
    #[arg(value_name = "certs_dir", required = true)]
    pub certs_dir: PathBuf,
}
```

Nit pick, but do we want to match the value name with the variable name?